readthedocs/rtd_tests/utils.py

```diff
@@ -89,12 +89,24 @@ def make_test_git():
 
     # Checkout to master branch again
     check_output(['git', 'checkout', 'master'], env=env)
-
-    # Create some tags
     chdir(path)
     return directory
 
 
+def create_tag(directory, tag, annotated=False):
```

Comment:
`create_git_tag` maybe?

and I think that `directory` can be renamed to `path` also, to keep the same wording.

Reply:
Directory makes reference to the tmp directory, and path is used for the cwd,